src/main/kotlin/com/openlattice/organizations/HazelcastOrganizationService.kt

```diff
@@ -170,6 +176,10 @@ class HazelcastOrganizationService(
         }) as Set<UUID>
     }
 
+    fun getOrganizationDatabaseName(organizationId: UUID): String {
+        return organizationDatabases.getValue(organizationId).name
```

Comment:
How big is the return value here? If large consider using offloadable entry processor to return just the name.

Reply:
It's an OrganizationDatabase object (defined in api) which is just an int and a String